discovery-provider/alembic/trigger_sql/handle_save.sql

```diff
@@ -41,7 +44,9 @@ begin
         AND r.save_type = new.save_type
     )
     where user_id = new.user_id;
-
+  	if new.is_delete IS FALSE then
+		  select tracks.owner_id, tracks.remix_of into owner_user_id, track_remix_of from tracks where is_current and track_id = new.save_item_id;
```

Comment:
looks like handle_save has a whole provision for `remix_of` notification... should repost have the same thing?

Why exclusive to save?

Reply:
this is on purpose for "co-sign" notifications where it notifies the artist if the original track's artist likes the remixed track